modules/support-gather-data.adoc

```diff
@@ -27,6 +27,29 @@ $ oc adm must-gather
 ====
 If this command fails, for example if you cannot schedule a Pod on your cluster, then use the `oc adm inspect` command to gather information for particular resources. Contact Red Hat Support for the recommended resources to gather.
 ====
++
+[NOTE]
+====
+If your cluster is using a restricted network you must import the default must-gather image, or mirror the latest must-gather image, and then run the `oc adm must-gather` command.
+
+Importing the default must-gather image:
+
+----
+oc import-image is/must-gather -n openshift
+
+oc adm must-gather
+----
+
+Mirroring the latest must-gather image:
+
+----
+oc image mirror quay.io/openshift/origin-must-gather:latest \
```

Comment:
All of these commands shouldn't be in a note. Please separate these into different sections. I think that the instructions to mirror the content should go in https://docs.openshift.com/container-platform/4.2/installing/install_config/installing-restricted-networks-preparations.html and that mirroring the content should be a prereq.